packages/components/carousel/src/__tests__/Carousel.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from '../../../../../tests/test-utils';
+import Carousel from '../Carousel';
+
+const mockContents = [
+  { src: 'image1.jpg', alt: 'Image 1' },
+  { src: 'image2.jpg', alt: 'Image 2' },
+  { src: 'video1.mp4', type: 'video/mp4' },
+  { html: '<div>HTML Content</div>' },
+];
+
+describe('Carousel Component', () => {
```

Comment:
The tests are useless. We don't need to test the type of content of each item. The main test must be clicking on an arrow and check if the position selected is correct!